src/test/java/io/github/perrymant/moneymaker/FileReaderTest.java

```diff
@@ -10,6 +10,30 @@
     public void canDisplayHelpInfo() {
         assertEquals("Some Text", new FileReader().read("flatfile.txt"));
     }
+
+    @Test
+    public void canDisplay_report_seed_data_json() {
```

Comment:
We've already got this test case covered in the test above. No need to add this one.